.circleci/config.yml

```
@@ -240,18 +240,33 @@ jobs:
     steps:
       - checkout_and_fetch_tags
       - restore_node_modules
+      - add_ssh_keys:
+          fingerprints:
+            - "89:ed:56:33:f5:1b:7b:b6:a1:04:02:a0:f9:47:13:0a"
+      - run:
+          name: 'Grab Current Version Number'
+          command: echo "export CURRENT_VERSION=$(npm run --silent get-current-version | xargs)" >> $BASH_ENV
+      - run:
+          name: 'Build Docs'
+          command: npm run build:docs
+      - run:
+          name: 'Publish Docs'
+          command: |
+            # Check if there's anything to commit before committing
+            if [ -n "$(git status --porcelain)" ]; then
+              git add docs/
+              git commit -m "docs(): update docs"
```

Can we move this after the version bump so our commit message would look something like 
```suggestion
              git commit -m "docs(API): update docs for v${VERSION}"
```

---

I don't want to put a version on it, if in the case that a dev supplies a `[skip npm]` commit

---

I don't want to put a version on it, in case a dev supplies a `[skip npm]` commit